### ISSUE TITLE
Negative COVID test results to also be ingested

### ISSUE THREAD

**Issue by z023** [globaldothealth/list#1047]
Could cases whose COVID test results are negative also be ingested? E.g. Estonia source data contains positive and negative test results (https://opendata.digilugu.ee/opendata_covid19_test_results.csv).

**Comment by attwad** [comment]
Not until the schema supports it: https://github.com/globaldothealth/list/issues/457

**Comment by allysonjp715** [comment]
Duplicate of #457
